src/cartogram_info/read_csv.cpp

```diff
@@ -70,39 +70,104 @@ void CartogramInfo::read_csv(const argparse::ArgumentParser &arguments)
     // Get target area
     csv::CSVField area_field = row[area_col];
     double area;
+    std::string area_as_str = area_field.get();
+    char comma = ',';
+    char decimal = '.';
+    long unsigned int string_size = area_as_str.length();
```

Comment:
It is not a big deal but it would look nicer if we use a more meaningful variable name such as `area_str_size`.

Also, we can expect the area string to be fairly short, so using type `int` instead of `long unsigned int` would suffice.